easybuild/tools/toolchain/craytoolchain.py

```diff
@@ -0,0 +1 @@
+craytoolchain.py-with-swap-fix-gpp
```

Comment:
I created this symlink for our production installation in order to avoid overwriting the stock craytoolchain.py with our modified version (that it is still under testing).

I suppose that craytoolchain.py should come from PR [#2222](https://github.com/hpcugent/easybuild-easyconfigs/pull/2222) and the file+symlink could be removed from this PR.